config.js

```diff
@@ -19,7 +19,7 @@ const defaultConfig = {
       'owners.name',
     ],
     attributesForFaceting: ['onlyFilter(name)', 'keywords'], // optionalFacetFilters to boost the name
-    customRanking: ['desc(downloadsLast30Days)'],
+    customRanking: ['desc(downloadsLast30Days)', 'desc(dependents)'],
```

Comment:
It's not exactly the same, but something with a lot of dependents will also get downloaded more